pyanaconda/kickstart.py

```diff
@@ -1070,6 +1070,19 @@ def execute(self, *args):
             logger.updateRemote(remote_server)
 
 class Network(commands.network.RHEL7_Network):
+    def parse(self, args):
+        nd = commands.network.RHEL7_Network.parse(self, args)
+        setting_only_hostname = nd.hostname and len(args) <= 2
```

Comment:
What are the two args that could be specified here? AFAICT, *network --hostname=blah* has no args.